apps/src/templates/teacherDashboard/teacherSectionsRedux.js

```diff
@@ -972,7 +978,9 @@ export const sectionFromServerSection = serverSection => ({
   studentCount: serverSection.studentCount,
   code: serverSection.code,
   courseId: serverSection.course_id,
-  scriptId: serverSection.script ? serverSection.script.id : null,
+  scriptId: serverSection.script
+    ? serverSection.script.id
+    : serverSection.script_id || null,
```

Comment:
It's tricky to tell from this Revert Revert, but the problem with the tests  was that null does not equal undefined. The change above on line 983 and reversing below from #31578 
<img width="1067" alt="Screen Shot 2019-10-29 at 9 35 16 PM" src="https://user-images.githubusercontent.com/12300669/67829368-2555e200-fa94-11e9-97b8-794240d00406.png">